src/stores/A_store.js

```diff
@@ -60,15 +63,81 @@ class A_store extends ReduceStore {
             }
 
             case ActionTypes.RESET: {
-                return {pathGrid: new Grid(SIZE, SIZE), player: {x: -1, y: -1}, end: {x: -2, y: -2}, stage: 'STARTP', path: ''};
+                return {pathGrid: new Grid(SIZE, SIZE), 
```

Comment:
generally multiline objs are styled as
```js
{
  k: 'v',
  trailing: 'comma',
}
```

we're already doing this in getInitialState so please be consistent